src/controllers/levelcontroller.js

```diff
@@ -41,6 +41,8 @@ class LevelController extends Controller
             showConfirmButton: false
         });
 
+        (new FactoryController)._updateFactory();
```

Comment:
But then it would update the factory every day, even when the level has not been completed. E.g. when you're placing on order at the end of the day, it might get wiped when the factory refreshes.